voluptuous_serialize/__init__.py

```diff
@@ -12,12 +12,17 @@
 }
 
 
-def convert(schema):
+def convert(schema, custom_serializer=None):
```

Comment:
Make it a keyword arg only
```suggestion
def convert(schema, *, custom_serializer=None):
```